brats/data/datasets.py

```diff
@@ -50,3 +56,17 @@ def __len__(self) -> int:
 
     def __getitem__(self, idx: int) -> typing.Tuple[typing.Any, ...]:
         return tuple(dataset[idx] for dataset in self.datasets)
+
+
+def read_dataset_json(path_to_json):
+    """
+    Reads pairs of images and masks from json file.
+    :param path_to_json: Path to the file from decathlon challange
+    :return: Tuple with list of paths to images and list of path to masks
+    """
+    with open(path_to_json, "r") as json_file:
+        json_dict = json.load(json_file)
+    root = os.path.dirname(path_to_json)
+    images_paths = [os.path.join(root, line["image"].replace("./", "")) for line in json_dict["training"]]
```

Comment:
Will this always be `"training"`? Maybe provide it as a default parameter?

Reply:
yeah, sometimes we would like to read test files, I forgot to add it. Fixed